src/Bicep.Core/CoreResources.resx

```diff
@@ -434,4 +434,10 @@
   <data name="DecompilerImperfectionsRule_Variable" xml:space="preserve">
     <value>The name of variable '{0}' appears to have originated from a naming conflict during a decompilation from JSON.</value>
   </data>
+  <data name="MaxNumberAssertsRuleDescription" xml:space="preserve">
+    <value>Maximum number of 'assert' statements used.</value>
+  </data>
+  <data name="MaxNumberAssertsRuleMessageFormat" xml:space="preserve">
+    <value>Too many predeployment conditions. Number of 'assert' statements is limited to {0}.</value>

```

Comment:
What is a "predeployment condition"?

Reply:
It's a suggested phrase in place of assertions. I'm treating it as a placeholder before we have a new keyword to replace "assert".